tests/filecheck/dialects/linalg/linalg_ops.mlir

```diff
@@ -48,6 +48,9 @@ linalg.generic {indexing_maps = [affine_map<(d0, d1) -> ()>, affine_map<(d0, d1)
  %fill = linalg.fill ins(%0 : f32) outs(%t3 : tensor<4x16xf32>) -> tensor<4x16xf32>
  linalg.fill ins(%0 : f32) outs(%m3 : memref<4x16xf32>)
 
+ %copy = linalg.copy ins(%t1 : tensor<4x16xf32>) outs(%t3 : tensor<4x16xf32>) -> tensor<4x16xf32>
```

Comment:
can you please add an attribute dictionary here? something like `{my_attr}` in the right place?